CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.8.0
```

Comment:
0.7.0?

Reply:
There is a `0.7.0-alpha.1` (dev) version published to so I thought maybe this could be 0.8.0. But could be 0.7.0. No strong opinion on this.